tools/subgraph-example/subgraph/tests/example-hts.test.ts

```diff
@@ -0,0 +1,60 @@
+import {
```

Comment:
Missing license header here also. SHould check all the `.js` and `.ts` files have them

Reply:
Added to all missing files.